io.github.chtholine.steganograpp.yml

```diff
@@ -0,0 +1,21 @@
+id: io.github.chtholine.steganograpp
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: steganograpp
+finish-args:
+ - --socket=fallback-x11
+ - --socket=wayland
+ - --share=ipc
+ - --filesystem=host
+modules:
+ - name: steganograpp
+   buildsystem: simple
+   build-commands:
+     - chmod +x steganograpp/io.github.chtholine.steganograpp
+     - install -D steganograpp/io.github.chtholine.steganograpp /app/bin/steganograpp
+   sources:
+      - type: file
+        url: https://github.com/chtholine/steganograpp/raw/main/dist/io.github.chtholine.steganograpp
```

Comment:
What happened to aarch64?

```
io.github.chtholine.steganograpp: ELF 64-bit LSB executable, x86-64, version 1 (SYSV)
```

Reply:
Also why did you upload a binary to git?

Reply:
Hello! I'm new to this and appreciate your assistance. I used PyInstaller to make an executable out of the python file with dependencies and decided to upload it. Should I do it other way?

Reply:
You licensed it with GPL-3.0, so just build it here and upload the source to the github repo.

Reply:
also you never link from a branch but always from a commit.